Sources/App/Features/Home/HomeRouteController.swift

```diff
@@ -165,14 +165,14 @@ struct HomeRouteController: RouteCollection {
     }
     
     private func getPhase(req: Request, event: Event) throws -> Phase {
-        let isPreviousEvent = event.date <= Date() &&
-            // This is a little trick where we can set the data to anything older than 2015 to hide date and tickets
-            event.date.timeIntervalSince1970 > 1420074000
+        // This is a little trick where we can set the data to anything older than 2015 to hide date and tickets
+        let isntHiddenEvent = event.date.timeIntervalSince1970 > 1420074000
```

Comment:
```suggestion
        let isEventHidden = event.date.timeIntervalSince1970 > 1420074000
```
or similar?

Reply:
It's literally the inverse, but I'll tweak so it reads better 😂